packages/eslint-config-react-app/package.json

```diff
@@ -17,9 +17,9 @@
     "jest.js"
   ],
   "peerDependencies": {
+    "@babel/eslint-parser": "^7.13.0",
```

Comment:
Do you mind updating the readme alongside this to update any references to `babel-eslint`?